packages/devtools_app/lib/src/provider/provider_screen.dart

```diff
@@ -0,0 +1,115 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:provider/provider.dart' as provider show Provider;
+
+import '../banner_messages.dart';
+import '../screen.dart';
+import '../split.dart';
+import './instance_viewer/instance_details.dart';
+import './instance_viewer/instance_providers.dart';
+import './instance_viewer/instance_viewer.dart';
+import 'provider_list.dart';
+
+final _hasErrorProvider = Provider.autoDispose<bool>((ref) {
+  if (ref.watch(providerIdsProvider) is AsyncError) return true;
+
+  final selectedProviderId = ref.watch(selectedProviderIdProvider).state;
+
+  if (selectedProviderId == null) return false;
+
+  final instance = ref.watch(
+    rawInstanceProvider(InstancePath.fromProviderId(selectedProviderId)),
+  );
+
+  return instance is AsyncError;
+});
+
+class ProviderScreen extends Screen {
+  const ProviderScreen()
+      : super.conditional(
+          id: id,
+          requiresLibrary: 'package:provider/',
```

Comment:
I assume this doesn't work on all versions of package:provider. What versions does it work on and what message do we show if it is not supported?
You could use EvalOnDartLibrary to verify the provider library has a required method only available on versions you support.

Reply:
The version supported is ^5.0.0

There is a banner that shows if any evaluation related to `package:provider` throws an error, which suggests to use ^5.0.0